### PR TITLE
Adding oraclecloudapps.com from Oracle Autonomous Database

### PR DESCRIPTION
**Pull request by pmandala**
Public Suffix List (PSL) Pull Request (PR) Template
====

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

__Submitter affirms the following:__ 

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Oracle Corporation is an American multinational computer technology company headquartered in Austin, Texas. In 2020, Oracle was the third-largest software company in the world by revenue and market capitalization.

Organization Website: https://www.oracle.com/
https://www.oracle.com/autonomous-database/

Reason for PSL Inclusion
====

Oraclecloudapps,com, oraclegovcloudapps.com and oraclegovcloudapps.uk are for customer use when using the Oracle Cloud Infrastructure platform for Automonous Database. These domains are used for customers accessing oracle application deployed in against their database. 

Customer are reported Cookie Tossing issues hence we are fixing to deliver Customer web content must only be accessible via domains in the Public DNS Suffix List. 

Number of users this request is being made to serve: In millions

DNS Verification via dig
=======

```
dig +short TXT _psl.oraclecloudapps.com
"https://github.com/publicsuffix/list/pull/2130"
```

```
dig +short TXT _psl.oraclegovcloudapps.com
"https://github.com/publicsuffix/list/pull/2130"
```

```
dig +short TXT _psl.oraclegovcloudapps.uk 
"https://github.com/publicsuffix/list/pull/2130"
```

Results of Syntax Checker (`make test`)
=========

```
========================================================================
Testsuite summary for libpsl 0.21.5
========================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
========================================================================
```

### CODE REVIEW

**Comment by pmandala** [comment]
Original PR

**Review by simon-friedberger — CHANGES_REQUESTED** [review]
Sorry if we were not clear. The request is that you only have a single section not that you sort the two sections next to each other.

**Comment by pmandala** [comment]
Resubmitting tests 

```============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in msvc

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12520,13 +12520,17 @@ on.crisp.email
...
 // Submitted by Gregory Drake <support@dyn.com>
 // Note: This is intended to also include customer-oci.com due to wildcards implicitly including the current label
 *.customer-oci.com
 *.oci.customer-oci.com
 *.ocp.customer-oci.com
 *.ocs.customer-oci.com
+*.oraclecloudapps.com
+*.oraclegovcloudapps.com
+*.oraclegovcloudapps.uk
+
```
Comment (wdhdev) [inline public_suffix_list.dat:12533]: Please remove this extra line

File: public_suffix_list.dat [hunk 2]
```diff
@@ -12520,13 +12520,17 @@ on.crisp.email
 // Submitted by Marvin Wiesner <Marvin@curv-labs.de>
 curv.dev
 
-// Customer OCI - Oracle Dyn : https://cloud.oracle.com/home https://dyn.com/dns/
+// Oracle Dyn https://cloud.oracle.com/home https://dyn.com/dns/
```
Comment (wdhdev) [inline public_suffix_list.dat:12523]:
```suggestion
// Oracle Dyn : https://cloud.oracle.com/home https://dyn.com/dns/
```

File: public_suffix_list.dat (hunk 1 above)
Comment (pmandala) [reply public_suffix_list.dat:12533]: Removed extra line

**Comment by pmandala** [comment]
Resubmitting tests

```============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in msvc

**Comment by wdhdev** [comment]
@pmandala Please update the _psl TXT records to this PR, thanks.

**Comment by pmandala** [comment]
@wdhdev Will you merge a brach with multiple commits?  if i rebase again with force push the PR cannot be reopened again like last time. I need to open another new PR again.  if that is case i better rebase and update _psl TXT for the new PR instead of this PR.

**Comment by simon-friedberger** [comment]
@pmandala Hm...Updating the DNS records wouldn't need a new branch but yes, we take PRs with multiple commits but actually force pushing to a branch doesn't mean that your PR gets closed. Maybe you did something else?

**Comment by simon-friedberger** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [x] oraclecloudapps.com expires 2033-06-17
  * [x] oraclegovcloudapps.com expires 2033-08-08
  * [x] oraclegovcloudapps.uk expires 2033-10-09
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [x] Sorting
* [x] Reasoning/Organization description
* [x] Non-personal email address

**Comment by simon-friedberger** [comment]
@pmandala Please remind your colleagues of the requirements.

**Pull request merged**